Makefile

```diff
@@ -300,6 +300,7 @@ bench-daily:
 	go test -tags intest github.com/pingcap/tidb/pkg/expression -run TestBenchDaily -bench Ignore --outfile bench_daily.json
 	go test -tags intest github.com/pingcap/tidb/pkg/util/rowcodec -run TestBenchDaily -bench Ignore --outfile bench_daily.json
 	go test -tags intest github.com/pingcap/tidb/pkg/util/codec -run TestBenchDaily -bench Ignore --outfile bench_daily.json
+	go test -tags intest github.com/pingcap/tidb/pkg/statistics/handle/cache -run TestBenchDaily -bench Ignore --outfile bench_daily.json
```

Comment:
The bench test looks poor written.
When I run it on my local machine:

![image](https://github.com/pingcap/tidb/assets/1420062/5b65494b-e6a3-4722-99c0-4108ae3f97fb)

Reply:
let me check it.